### ISSUE TITLE
Could not autoload puppet/provider/file_concat/posix

### ISSUE THREAD

**Issue by thlapin** [electrical/puppet-lib-file_concat#24]
Using rspec-puppet to test my puppet module which uses the logstash module which depends on the file_concat module. For the first test I see the following error:

```
     Failure/Error: it { is_expected.to compile.with_all_deps }
       error during compilation: Could not autoload puppet/type/file_concat: Could not autoload puppet/provider/file_concat/posix: cannot load such file -- puppet/provider/file_concat on node
```

This appears to have been introduced in 143eed41e57c78aa18360a21d108c97512f31979.

**Comment by electrical** [comment]
That's weird, haven't seen that error so far.
The acceptance tests that i run have not exposed this.
We are in the process of chancing allot to the code with PR #23 to make it simpler to manage the library.

**Comment by thlapin** [comment]
I'm also not seeing it in acceptance tests using beaker.

I've just modified fixtures to use bmjen:add-new-params and the problem has gone away. So the refactoring there appears to be contain the necessary fix.

**Comment by electrical** [comment]
We will be merging the PR fairly soon and will do a release then.

**Comment by electrical** [comment]
Hi,

I just released version 1.0.0 at https://forge.puppetlabs.com/electrical/file_concat/

This should solve the issue you had.

**Comment by thlapin** [comment]
Indeed, 1.0.0 fixes this issue. Thanks!
